applications/MeshingApplication/tests/test_MeshingApplication.py

```diff
@@ -1,28 +1,28 @@
 # import Kratos
 import KratosMultiphysics
-import KratosMultiphysics.ExternalSolversApplication
-import KratosMultiphysics.FluidDynamicsApplication
-import KratosMultiphysics.MeshingApplication
+import KratosMultiphysics.ExternalSolversApplication as ExternalSolversApplication
+import KratosMultiphysics.FluidDynamicsApplication   as FluidDynamicsApplication
+import KratosMultiphysics.MeshingApplication         as MeshingApplication
 
 # Import Kratos "wrapper" for unittests
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
 # Import the tests o test_classes to create the suits
 ## SMALL TESTS 
-from test_refine import TestRedistance as TTestRedistance
-from SmallTests import TwoDHessianTest as TTwoDHessianTest
-from SmallTests import ThreeDHessianTest as TThreeDHessianTest
-from SmallTests import TwoDCavityTest as TTwoDCavityTest
-from SmallTests import CoarseSphereTest as TCoarseSphereTest
-from SmallTests import TwoDDynamicBeamTest as TTwoDDynamicBeamTest
-from SmallTests import ThreeDDynamicBeamTest as TThreeDDynamicBeamTest
-from SmallTests import TwoDDynamicPlasticBeamTest as TTwoDDynamicPlasticBeamTest
+from test_refine import TestRedistance                      as TTestRedistance
+from SmallTests  import TwoDHessianTest                     as TTwoDHessianTest
+from SmallTests  import ThreeDHessianTest                   as TThreeDHessianTest
+from SmallTests  import TwoDCavityTest                      as TTwoDCavityTest
+from SmallTests  import CoarseSphereTest                    as TCoarseSphereTest
+from SmallTests  import TwoDDynamicBeamTest                 as TTwoDDynamicBeamTest
+from SmallTests  import ThreeDDynamicBeamTest               as TThreeDDynamicBeamTest
+from SmallTests  import TwoDDynamicPlasticBeamTest          as TTwoDDynamicPlasticBeamTest
 
 ## NIGHTLY TESTS
-from NightlyTests import StanfordBunnyTest as TStanfordBunnyTest
+from NightlyTests import StanfordBunnyTest                  as TStanfordBunnyTest
```

Comment:
i would really like this test to be written in "standalone" python format instead of a test run created by the interface